scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/SchedulerServiceImpl.java

```diff
@@ -1391,6 +1391,11 @@ private int executeVoidFunction(BiFunction<RestClient, Integer, Void> action, Li
 
     private String executeFunctionReturnStreamAsString(Function<RestClient, InputStream> function)
             throws ServiceException, RestServerException {
+        return executeFunctionReturnStreamAsString(function, false);
+    }
+
+    private String executeFunctionReturnStreamAsString(Function<RestClient, InputStream> function, boolean keepNewLines)
+            throws ServiceException, RestServerException {
         RestClient restClientProxy = getRestClientProxy();
 
         InputStream inputStream = null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)